akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -234,7 +259,7 @@ private[http2] class Http2ServerDemux(http2Settings: Http2CommonSettings, initia
           case Setting(Http2Protocol.SettingIdentifier.SETTINGS_MAX_FRAME_SIZE, value) =>
             multiplexer.updateMaxFrameSize(value)
           case Setting(Http2Protocol.SettingIdentifier.SETTINGS_MAX_CONCURRENT_STREAMS, value) =>
-            debug(s"Setting max concurrent streams to $value (not enforced)")
+            debug(s"Setting max concurrent streams to $value (not respected)")
```

Comment:
I think `enforcing` applies to settings the local peer set and requires, while `respect` is what the local peer should do with remote settings.
This method applies remote settings and logs that "Hey, I received `SETTINGS_MAX_CONCURRENT_STREAMS` but I'm not respecting it".